Seems to be an extra argument in gvPluginList

I tried building libmediainfo 24.11 on Slackware 15.0 and ran into the following error:

```
libtool: compile:  g++ -DPACKAGE_NAME=\"libmediainfo\" -DPACKAGE_TARNAME=\"libmediainfo\" -DPACKAGE_VERSION=\"24.11\" "-DPACKAGE_STRING=\"libmediainfo 24.11\"" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DPACKAGE=\"libmediainfo\" -DVERSION=\"24.11\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ZLIB_H=1 -I. -I../../../Source -I../../../../ZenLib/Source -std=c++11 -O2 -fPIC -DUNICODE -DSIZE_T_IS_LONG -I/usr/include/graphviz -I/usr/local/include -I/usr/include/ -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I../../../Source/ThirdParty/aes-gladman -I../../../Source/ThirdParty/md5 -I../../../Source/ThirdParty/sha1-gladman -I../../../Source/ThirdParty/sha2-gladman -I../../../Source/ThirdParty/hmac-gladman -I../../../Source/ThirdParty/base64 -DUNICODE -fPIC -O2 -fvisibility=hidden -fvisibility-inlines-hidden -MT ../../../Source/MediaInfo/Image/File_Psd.lo -MD -MP -MF ../../../Source/MediaInfo/Image/.deps/File_Psd.Tpo -c ../../../Source/MediaInfo/Image/File_Psd.cpp  -fPIC -DPIC -o ../../../Source/MediaInfo/Image/.libs/File_Psd.o
../../../Source/MediaInfo/Export/Export_Graph.cpp: In function ‘ZenLib::Ztring MediaInfoLib::Dot2Svg(const ZenLib::Ztring&)’:
../../../Source/MediaInfo/Export/Export_Graph.cpp:133:34: error: too many arguments to function ‘char** gvPluginList(GVC_t*, const char*, int*)’
  133 |     char** Renderers=gvPluginList(Context, "render", &Renderers_Size, NULL);
      |                      ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../../../Source/MediaInfo/Export/Export_Graph.cpp:35:
/usr/include/graphviz/gvc.h:120:16: note: declared here
  120 | GVC_API char **gvPluginList(GVC_t *gvc, const char *kind, int *sz);
      |                ^~~~~~~~~~~~

```

Upon looking at the commit history for [Source/Mediainfo/Export/Export_Graph.cpp](https://github.com/MediaArea/MediaInfoLib/blob/master/Source/MediaInfo/Export/Export_Graph.cpp), I found the 594c6f4 commit. In looking at the [gvPluginList function](https://github.com/MediaArea/MediaInfoLib/blob/ef5750b22296e33ea6e6254ea3ab895853c089bb/Source/MediaInfo/Export/Export_Graph.cpp#L133) and comparing it to the [typedef line](https://github.com/MediaArea/MediaInfoLib/blob/594c6f4221009489773d1ef880e056ab8a7bf2e1/Source/MediaInfo/Export/Export_Graph_gvc_Include.h#L110C1-L110C133) in [Export_Graph_gvc_Include.h](https://github.com/MediaArea/MediaInfoLib/blob/master/Source/MediaInfo/Export/Export_Graph_gvc_Include.h) that was added in the same commit, it seems it gvPluginList should only have 3 arguments, not 4 (or the typedef line should be changed to have 4 arguments, but I am not much of a programmer to know better). When I removed the NULL argument, the source was able to compile successfully.

Since Slackware uses a bash script to build packages, I simply added a sed line to the script to remove the NULL argument.

`sed -i 's|Renderers_Size, NULL|Renderers_Size|' Source/MediaInfo/Export/Export_Graph.cpp`

MediaInfo itself was then updated to 24.11 and all seem to work fine with several test files using both the CLI and GUI.

--- a/Source/MediaInfo/Export/Export_Graph.cpp
+++ b/Source/MediaInfo/Export/Export_Graph.cpp
@@ -130,7 +130,7 @@ Ztring Dot2Svg(const Ztring& Dot)
         return ToReturn;
 
     int Renderers_Size=0;
-    char** Renderers=gvPluginList(Context, "render", &Renderers_Size, NULL);
+    char** Renderers=gvPluginList(Context, "render", &Renderers_Size);
     for (int Pos=0; Pos<Renderers_Size; Pos++) {
         if (!strcmp(Renderers[Pos], "cairo"))
             Cairo=true;

--- a/Source/MediaInfo/Export/Export_Graph_gvc_Include.h
+++ b/Source/MediaInfo/Export/Export_Graph_gvc_Include.h
@@ -107,7 +107,7 @@ extern "C"
 // Interface
 typedef GVC_t* (*gvc_Module_gvContext)(); static gvc_Module_gvContext gvContext=NULL;
 typedef int (*gvc_Module_gvFreeContext)(GVC_t*); static gvc_Module_gvFreeContext gvFreeContext=NULL;
-typedef char** (*gvc_Module_gvPluginList)(GVC_t*, const char*, int*, const char*); static gvc_Module_gvPluginList gvPluginList=NULL;
+typedef char** (*gvc_Module_gvPluginList)(GVC_t*, const char*, int*); static gvc_Module_gvPluginList gvPluginList=NULL;
 typedef int (*gvc_Module_gvLayout)(GVC_t*, graph_t*, const char*); static gvc_Module_gvLayout gvLayout=NULL;
 typedef int (*gvc_Module_gvFreeLayout)(GVC_t*, graph_t*); static gvc_Module_gvFreeLayout gvFreeLayout=NULL;
 typedef int (*gvc_Module_gvRenderData)(GVC_t*, graph_t*, const char*, char**, unsigned int*); static gvc_Module_gvRenderData gvRenderData=NULL;

